torch/testing/_internal/common_methods_invocations.py

```diff
@@ -13144,27 +13160,18 @@ def generate_std_var_kwargs(t: torch.Tensor, **kwargs):
                     supports_fwgrad_bwgrad=True,
                     assert_autodiffed=True,
                     supports_two_python_scalars=True,
-                    # TODO: FIXME: pow needs a way of specifying that for integer
-                    #   types only it does not support negative exponentes
-                    rhs_make_tensor_kwargs=dict(low=1),
-                    skips=(
-                        # nan vs nan comparisons
-                        # https://github.com/pytorch/pytorch/issues/74279
-                        DecorateInfo(unittest.skip("Skipped!"), 'TestGradients'),
-                    )),
+                    sample_inputs_func=sample_inputs_pow),
```

Comment:
It would be nice to have the `.ref` field added in this PR. Maybe we could add skips specifically for the cases that are failing?